docs/examples/add-to-your-application/python/app/routes.py

```diff
@@ -0,0 +1,30 @@
+from flask import Flask
+from oso import NotFoundError
+from .models import users_db, Repository
+from .oso import oso
+
+
+app = Flask(__name__)
+
+def serialize(r):
+    return str(r)
+
+# implemented here to not polute code samples in model.md
+class User:
+    @staticmethod
+    def get_current_user():
+        return users_db['larry']
+
+# docs: begin-show-route
+@app.route("/repo/<name>")
+def repo_show(name):
+    repo = Repository.get_by_name(name)
```

Comment:
This seems low priority, but this might not be sending the right message about how to handle errors.

The `ForbiddenError` is not handled currently, and In theory `Repository.get_by_name` could throw it's own version of Notfound, right?

Reply:
Hm yeah that's true. There is more complete error handling information in enforce authorization, which I link to. This isn't complete because it doesn't handle the `ForbiddenError`. I could just handle `AuthorizationError` here, but thats also not completely correct.

Trying to balance keeping the code simple & completely correct.